assembl/templates/system/assembl.service.jinja2

```diff
@@ -0,0 +1,27 @@
+# Sample supervisord service file
```

Comment:
why is this file a Jinja2 template, there is nothing to be changed in it. The values are all set

Reply:
They're jinja2 files because for the moment they are static, but in near future, they will be variable. Trying to keep things consistent in case changes are required.